client-v2/src/shared/components/CollectionItemMetaContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I've broken out a lot of the Article specific styled components into CollectionItem components to share with the new SnapLink component.